plugins/system/file_length_

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+: << =cut
+
+=head1 NAME
+
+file_length_ - Plugin to monitor the length of specified files
+
+Useful for things such as lists (white, black, user, ...).
+
+=head1 CONFIGURATION
+
+    [file_length_IDENTIFIER]
+    env.files FILEPATHGLOB1 FILEPATHGLOB2 ...
+    env.category DEFAULTS_TO_system
+    env.title OPTIONAL_TITLE
+    env.logarithmic 1
+
+=head1 AUTHOR
+
+Olivier Mehani (based on disk/log_sizes)
+
+=head1 LICENSE
+
+GPLv2
+
+=cut
+
+#NAME=`echo $0 | sed 's/.*_//'`
+NAME=${0#*_}
+TITLE=${title:-File lengths for $NAME}
+CATEGORY=${category:-system}
+
+FILES=${files:-/var/log/messages}
+FILES=$(echo $(ls $FILES))
```

Comment:
`echo $FILES` should be sufficient for globbing (if this your point). Please add a comment regarding the purpose of this line.